### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by mflaxman**
I didn't test this and just did it from memory, but it's probably better than nothing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+### OSX ###
+```
+$ brew install bitcoin
+$ brew services start bitcoin
+```
+(more complicated instructions to build from source [here](https://github.com/bitcoin/bitcoin/blob/master/doc/build-osx.md))
+
+### Ubuntu ###
+TODO
```
Comment (jnewbery) [inline README.md]: `sudo add-apt-repository ppa:bitcoin/bitcoin && sudo apt-get update` should do it (https://launchpad.net/~bitcoin/+archive/ubuntu/bitcoin)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+### Install Dependencies ###
+
+Create and activate your virtual environment (not strictly required):
+```
+$ python3 -m virtualenv .venv3 && virtualenv .venv3/bin/activate
```
Comment (jnewbery) [inline README.md]: I'd prefer venv over virtualenv if running modern Python (https://launchpad.net/~bitcoin/+archive/ubuntu/bitcoin)

File: README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# A Jupyter notebook demonstrating the Bitcoin Test Framework #
+
```
Comment (jnewbery) [inline README.md:2]: I think we should have a warning here. Something like:

Do not run test instances of bitcoind on the same machine that you store your Bitcoin private keys. This test framework shouldn't interfere with your standard bitcoind directory, but why risk it?

**Review by jnewbery — COMMENTED** [review]
Thank you! Looks good. I've added a few comments to make it a bit more comprehensive.

**Comment by mflaxman** [comment]
Thanks @jnewbery those were all great points. Apologies for the delay.

Anyway, this should be good to go now! I'm not sure if github has a squash & merge button (you might have to pay for that), happy to rebase and collapse it down to one commit if you prefer.

**Comment by jnewbery** [comment]
Mega. Thanks Michael!

**Pull request merged**